### ISSUE TITLE
Support for more emoji!

### ISSUE THREAD

**Issue by daknob** [makersphereHQ/emoji-passcodes#1]
Here's a table of the emoji password length and the rounded equivalent length of a `[a-zA-Z0-9]` password:

| Emoji Password Length | Equivalent ASCII Password Length |
| --- | --- |
| 4 | 7 |
| 5 | 9 |
| 6 | 11 |
| 7 | 13 |
| 8 | 14 |
| 9 | 16 |
| 10 | 18 |

I would personally recommend a 7+ emoji long password since there exist formats where cracking an 8-character password can take as low as 6 hours in a single computer, much less in a cluster of computers.

As a bonus, 7 emojis can create much better stories that just 4!

**Comment by makersphereLabs** [comment]
Yep! We should add some kind of _advanced mode_ with more emoji inputs.
What about letting the users decide, how long their story should be. :smile:
We could define a min. number of inputs so users can add as many **additional emoji inputs** as they wish.

**Comment by taryarevontulet** [comment]
It'd be cool to support emojis variations like 🎅🏻🎅🏽🎅🏿 (different shades of Santa)

**Comment by taryarevontulet** [comment]
Tested on mobile with iOS 10.0.2 and the new gender icons (like male dancers 👯‍♂️, female weight lifters 🏋‍♀️, female surfer 🏄‍♀️ and similar) aren't working 😥
